packages/ui/src/machines/authenticator/actors/signIn.ts

```diff
@@ -399,10 +407,9 @@ export const signInActor = createMachine<SignInContext, AuthEvent>(
     },
     services: {
       async signIn(context) {
-        const source = context.formValues;
-        const { country_code, username, password } = source;
+        const { username, password } = context.formValues;
```

Comment:
Should this be 
```suggestion
        const { username, password } = event.data ?? context.formValues;
```
now that we're submitting form with data:

https://github.com/aws-amplify/amplify-ui/blob/9a10937632d79043ab79f62b5f932262bf7f7926/packages/react/src/components/Authenticator/SignIn/SignIn.tsx#L49-L55

Reply:
Ahh scratch that, https://github.com/aws-amplify/amplify-ui/blob/9a10937632d79043ab79f62b5f932262bf7f7926/packages/ui/src/machines/authenticator/actions.ts#L116-L119 resolves it